src/main/java/hw_13/CarDTO.java

```diff
@@ -0,0 +1,38 @@
+package hw_13;
+
+public class CarDTO {
+    private String model;
+
+    public int getCountOfWheels() {
+        return countOfWheels;
+    }
+
+    public void setCountOfWheels(int countOfWheels) {
+        if (countOfWheels < 4) {
+            throw new InvalidWheelCountException("The wheel count must be more than 3. Attempted to set: " + countOfWheels);
+        }
+
+        this.countOfWheels = countOfWheels;
+    }
```

Comment:
"getCountOfWheels()" and "setCountOfWheels(int countOfWheels)" should be located above "toString()" method but after constructor

In the very beginning of the class the variables should be located

Reply:
fixed